build.gradle

```diff
@@ -69,5 +69,5 @@ subprojects {
 }
 
 ext {
-    fluxCVersion = 'd4e583f21840111065f12e743cbf31de4307266b'
+    fluxCVersion = 'e940e07386a4ef413cf34f1f54ba2cb97593c7dc'
```

Comment:
Looking at the other PR, I think this should be `d82478c66e24cf6a50a714b7261a68c99ccf7145`, it's a bit confusing but did you rebase the FluxC branch?

Reply:
Good to go, @planarvoid 👍